backends/memcheck/ceed-memcheck-qfunction.c

```diff
@@ -63,8 +63,12 @@ static int CeedQFunctionApply_Memcheck(CeedQFunction qf, CeedInt Q, CeedVector *
     // Note: need field size because vector may be longer than needed for output
     CeedCallBackend(CeedQFunctionFieldGetSize(output_fields[i], &field_size));
     for (CeedSize j = 0; j < Q * field_size; j++) {
-      CeedCheck(!isnan(impl->outputs[i][j]), ceed, CEED_ERROR_BACKEND, "QFunction output %d entry %ld is NaN after restoring write-only access", i,
-                j);
+      char *kernel_path, *kernel_name;
+      CeedCallBackend(CeedQFunctionGetSourcePath(qf, &kernel_path));
+      CeedCallBackend(CeedQFunctionGetKernelName(qf, &kernel_name));
```

Comment:
Also not sure how much I like where this code is placed. But those char pointers are really only needed here.